### PR TITLE
[docker](readme) add readme for how to build doris image

### PR DESCRIPTION
**Pull request by catpineapple**
1.  add a readme for how to build doris image.
2. delete the docker/runtime/k8s directory, and no longer need native k8s resources to build a doris cluster.

### CODE REVIEW

**Comment by catpineapple** [comment]
run buildall

File: docker/runtime/README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## How to build a docker image
```
Comment (intelligentfu8) [inline docker/runtime/README.md:20]: build image

File: docker/runtime/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+### Preparation
```
Comment (intelligentfu8) [inline docker/runtime/README.md:22]: remove preparation
### Downlaod and copy not needed.

File: docker/runtime/README.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+### Preparation 
+
+#### Download and copy the Doris code repo
+
+```shell
+$ cd /to/your/workspace/
+$ git clone https://github.com/apache/doris.git
+$ cd doris/docker
+$ cp -r runtime/ /to/your/workspace/
+```
```
Comment (intelligentfu8) [inline docker/runtime/README.md:31]: remove this step

File: docker/runtime/README.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+   
+2. Unzip binary package
+
+```shell
+$ tar -zxvf apache-doris-2.1.5-bin-x64.tar.gz
+# or
+$ tar -zxvf apache-doris-2.1.5-bin-arm64.tar.gz
+```
+
+3. copy the antidote package
```
Comment (intelligentfu8) [inline docker/runtime/README.md:71]: extract archive

File: docker/runtime/README.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+You need to copy the corresponding directories to different files in sequence as shown in the table:
```
Comment (intelligentfu8) [inline docker/runtime/README.md:73]: different directories.

Comment (intelligentfu8) [inline docker/runtime/README.md:73]: below

File: docker/runtime/README.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+```
+
+#### Download, unzip and copy the antidote package
+
+Here we take the build of x64 (avx2) and arm64 platforms of 2.1.5 version as an example.
+
+1. Download binary
+
+Go to the doris [official website](https://doris.apache.org/download) to download the binary package you need, and pay attention to selecting the doris version and architecture type you need.
+
```
Comment (intelligentfu8) [inline docker/runtime/README.md:56]: Here we take the build of x64 (avx2) and arm64 platforms of 2.1.5 version as an example.

**Comment by catpineapple** [comment]
run buildall

**Review by zy-kkk — APPROVED** [review]
LGTM

**Review by intelligentfu8 — APPROVED** [review]

**Comment by catpineapple** [comment]
run buildall

**Comment by catpineapple** [comment]
run buildall

**Comment by catpineapple** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Review by zy-kkk — APPROVED** [review]

**Review by intelligentfu8 — APPROVED** [review]

**Pull request merged**